include/psa/crypto.h

```diff
@@ -3338,7 +3357,10 @@ psa_status_t psa_key_derivation_key_agreement(
  * stream.
  * The operation's capacity decreases by the number of bytes read.
  *
- * \param[in,out] operation The key derivation operation object to read from.
+ * If this function returns an error status, the operation enters an error
+ * state and must be aborted by calling psa_key_derivation_abort().
+ *
+* \param[in,out] operation The key derivation operation object to read from.
```

Comment:
Blocker: continuation comment lines must start with exactly one space and a `*`, otherwise this confuses one of our scripts (`tests/scripts/check-names.sh`).